cmd/juju/cloud/listcredentials_test.go

```diff
@@ -123,6 +127,27 @@ mycloud  me
 `[1:])
 }
 
+func (s *listCredentialsSuite) TestListCredentialsTabularInvalidCredential(c *gc.C) {
+	store := jujuclienttesting.WrapClientStore(s.store)
+	store.CredentialForCloudFunc = func(cloudName string) (*jujucloud.CloudCredential, error) {
+		if cloudName == "mycloud" {
+			return nil, errors.Errorf("expected error")
+		}
+		return s.store.CredentialForCloud(cloudName)
+	}
+	ctx := s.listCredentialsWithStore(c, store)
+	c.Assert(cmdtesting.Stdout(ctx), gc.Equals, `
+Cloud   Credentials
+aws     down*, bob
+azure   azhja
+google  default
+
+`[1:])
+	c.Assert(cmdtesting.Stderr(ctx), gc.Equals, `
```

Comment:
This implies the return code with be > 0 right?
Not sure that we want to have a non-success error code when the command ran successfully. We could add text to the output after the main tabular listing to print any bad credentials but maybe that's overkill.